var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -50,6 +55,9 @@ class QuantumEspresso(Package):
     # Versions of HDF5 prior to 1.8.16 lead to QE runtime errors
     depends_on('hdf5@1.8.16:+fortran', when='+hdf5')
 
+    # KISS for now and only build the mpi version
+    depends_on('mpi', when='+epw')
```

Comment:
I think it would be more useful to phrase this as a TODO like `TODO: add support for X in the future also`

Reply:
Fixed by 299e4d9